torchaudio/functional/functional.py

```diff
@@ -1403,6 +1410,8 @@ def resample(
         new_freq: float,
         lowpass_filter_width: int = 6,
         rolloff: float = 0.99,
+        resampling_method: str = "sinc_interpolation",
+        **kwargs,
```

Comment:
Can you check if `**kwargs` is supported by TorchScript? IIRC, it's not supported. That case, we need to think of a structured way to select window type and pass window parameter.